src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables for `firstLeg` and `secondLeg` and pass them on for better code readability.